packages/docxtractr/install

```diff
@@ -1,9 +1,14 @@
-
 #!/bin/bash
-
 set -x
 set -e
 
-add-apt-repository ppa:libreoffice/ppa
-apt-get update -qq
-apt-get install -y libreoffice
\ No newline at end of file
+OS_DISTRIBUTION=$(lsb_release -cs)
+if [ "${OS_DISTRIBUTION}" == "focal" ]; then
+  echo "docxtractr is not supported on Ubuntu 20.04 due to: https://github.com/rstudio/shinyapps-package-dependencies/issues/282"
+else
+  apt-get update -qq
+  apt-get install -y software-properties-common
```

Comment:
if this is added to the base shinyapps.io image (and the test image in this repo) then it would not be necessary in each install script